README.md

```diff
@@ -11,12 +11,17 @@ assets for use in probo builds.
 
 ## Usage
 
- 1. Create a bucket to hold assets.
- 2. Create a token that can be used to post assets into that bucket.
- 3. Upload an asset into a bucket with the upload token.
+ 1. Add Amazon S3 credentials.
+ 2. Create a bucket to hold assets.
+ 3. Create a token that can be used to post assets into that bucket.
+ 4. Upload an asset into a bucket with the upload token.
 
+### 1. Add Amazon S3 credentials defaults.config.yaml
```

Comment:
defaults.config.yaml shouldn't, generally, be editied - a new config file should be created and specified when invoking the app like so: `./bin/probo-asset-receiver -c my_config.yaml`